docs/Technical-Documentation/Architecture-Decision-Record/016-security-scan-waiver.md

```diff
@@ -22,6 +22,26 @@ When investigating, we sought out information on what other, modern ACF systems
 
 While we also investigated and developed technical solutions utilizing either React-app-rewired or Next.js, we decided that the departure from our existing stack was not a worthwhile investment and could lead to other complications down the road.
 
+### 2024 Update
+
+Another instance of this alert appeared in the Webinspect scans for January/February 2024, after the introduction of Kibana to our stack
+
+`HTML5: Misconfigured Content Security Policy` - `Recommendation: Remove the unsafe-eval and unsafe-inline values from the CSP directive values.`
+
+Referring to the following directives in our nginx config
+```
+set $CSP "${CSP}script-src 'self' 'unsafe-eval' 'unsafe-inline' http://{{env "KIBANA_BASE_URL"}}:5601;";
+set $CSP "${CSP}script-src-elem 'self' 'unsafe-inline' http://{{env "KIBANA_BASE_URL"}}:5601;";
+set $CSP "${CSP}script-src-attr 'self' 'unsafe-inline' http://{{env "KIBANA_BASE_URL"}}:5601;";
+```
+
+Kibana performs ajax requests from its frontend to scripts hosted on the server in order to provide styling and javascript functionality - in order to serve Kibana from behind a proxy, which is a requirement with cloud.gov, these `unsafe-inline` and `unsafe-eval` directives are required.
+We currently use Kibana version 7.4 [released October, 2019](https://www.elastic.co/blog/kibana-7-4-0-released). This requirement is present in our current version as mentioned by the Elastic team in
+* [elastic/kibana #27047](https://github.com/elastic/kibana/issues/27047#issuecomment-799680263) in March 2021
+* an [elastic discussion post](https://discuss.elastic.co/t/does-kibana-need-the-autorisation-of-unsafe-inline-or-unsafe-eval-to-work-properly/234390/2) in May 2020
+
+The CSP exceptions are fairly low risk as they are limited to artifacts within the Kibana domain.
```

Comment:
@jtimpe can you provide a couple of examples of what we mean by artifacts here? might also be worth explicitly stating there is no risk to data assets.

Reply:
open to ideas on how we can convey that there's no risk to data assets - what assets are specifically of concern?